spec.md

```diff
@@ -710,6 +710,11 @@ Note that per the [Orphans](#orphans) section, this error response does not
 cause orphan mitigation to be initiated. Therefore, Platforms receiving this
 error response SHOULD resend the request at a later time.
 
+Service Broker MAY choose to only support concurrent requests for certain
```

Comment:
this paragraph seems to be stating what is already true in the spec.  is it necessary?

Reply:
I agree. The spec already says `Service Brokers do not have to support concurrent requests that mutate the same resource.`. I like the addition of this behaviour clarification in general, but maybe this paragraph is redundant?

Reply:
True, but I wasn't sure if it was clear that a broker could be selective about it - meaning only support it on certain resources or even just for certain ops. Let me try to morph the existing sentence to add what I think is missing and see what you think...